packages/documentation/src/stories/components/avatar/avatar.docs.mdx

```diff
@@ -0,0 +1,31 @@
+import { Canvas, Controls, Meta } from '@storybook/blocks';
+import * as AvatarPictureStories from './avatar.stories';
+
+<Meta of={AvatarPictureStories} />
+
+<div className="docs-title">
+  # Avatar
+
+  <nav>
+    <link-design of={JSON.stringify(AvatarPictureStories)}></link-design>
+  </nav>
+</div>
+
+<p className="lead">Display a user's profile picture or their initials if the picture is unavailable.</p>
+
+The `<post-avatar>` component provides several properties to change its appearance. The most important one is probably the `email` property, which is used to request a Gravatar picture.
+
+Don't know Gravatar or haven't set up a profile yet, let's change this: [Signup](https://gravatar.com) now!
+
+<Canvas of={AvatarPictureStories.Default} />
```

Comment:
For some reason, the canvas has a different size when used with email address vs. without:

With avatar 130px
![image](https://github.com/user-attachments/assets/560f426b-26df-45c7-9a5c-91a3bbd0efd9)


Without gravatar 123px
![image](https://github.com/user-attachments/assets/bd441e0a-944f-4ccd-808d-a65c22d37c82)

The avatar retains it's size though, I don't know what the issue is. The only thing is that the layout shifts a bit when entering an email. Do you know what could cause this?